### PR TITLE
Add camera position example

### PR DESCRIPTION
**Pull request by akaszynski**
This PR extends #3010 by cleaning up the example and adding the additional `language = 'en'` needed to enable documentation caching.

Thanks @ENIAC-6!

### CODE REVIEW

File: pyvista/plotting/plotting.py [hunk 1]
```diff
@@ -1308,7 +1308,45 @@ def scale(self, scale):
...
+        >>> _ = pl.add_mesh(mesh, show_edges=True)
+        >>> pl.camera_position = 'xy'
+        >>> pl.show()
+
+        Set the camera position using a string and look at the ``'xy'`` plane.
+
+        >>> pl = pv.Plotter()
+        >>> _ = pl.add_mesh(mesh, show_edges=True)
+        >>> pl.camera_position = 'zy'
+        >>> pl.show()
```
Comment (adeak) [inline pyvista/plotting/plotting.py:1345]: Is one of these copies meant to be different, or is this just a duplicate by accident?

File: pyvista/plotting/plotting.py [hunk 2]
```diff
@@ -1308,7 +1308,45 @@ def scale(self, scale):
...
+
+        Set the camera position using a string and look at the ``'xy'`` plane.
+
+        >>> pl = pv.Plotter()
+        >>> _ = pl.add_mesh(mesh, show_edges=True)
+        >>> pl.camera_position = 'xy'
+        >>> pl.show()
+
+        Set the camera position using a string and look at the ``'xy'`` plane.
```
Comment (akaszynski) [inline pyvista/plotting/plotting.py:1340]:
```suggestion
        Set the camera position using a string and look at the ``'zy'`` plane.
```

File: pyvista/plotting/plotting.py (hunk 1 above)
Comment (akaszynski) [reply pyvista/plotting/plotting.py]: The example code

```py
>>> pl.camera_position = 'zy'
```

is different, but the text isn't. Fixed in 537b678.

Comment (adeak) [reply pyvista/plotting/plotting.py]: Ah, I missed that, thanks.

Comment (tkoyama010) [reply pyvista/plotting/plotting.py]: I am wondering if there is a better expression.

**Comment by akaszynski** [comment]
Merging this for a 0.35.2 patch release. Thanks for catching the docstring issue @adeak.

**Pull request merged**